test/TimeoutTest.php

```diff
@@ -8,6 +8,7 @@
 use Amp\Pause;
 use Amp\Success;
 use Amp\Promise;
+use function React\Promise\resolve;
```

Comment:
In a test… nah. I'd rather just use the fqn where it's called if you want to avoid ambiguity.